core/storage/base_model/gae_models.py

```diff
@@ -304,6 +304,17 @@ def update_timestamps_multi(cls, entities, update_last_updated_time=True):
         datastore_services.update_timestamps_multi(
             entities, update_last_updated_time=update_last_updated_time)
 
+    @classmethod
+    @transaction_services.run_in_transaction_wrapper
+    def put_multi_transactional(cls, entities):
+        """Stores the given datastore_services.Model instances.
```

Comment:
For here & the delete version above: please update the documentation to explain when to use this vs. the non-transactional versions. Given that these are part of the class's public API, we should try to make it clear to callers when to choose one versus the other.

Reply:
I won't be at home today for the most part, but I will get back to this asap

Reply:
@BenHenning referring to documentation, making the docstrings for the transactional version more intuitive should do the job right?

Reply:
Yes, I think what you added in the latest commits looks good minus one small bit that I left a separate comment on.